### PR TITLE
CORE-63 Added relative percentage change over time

### PR DESCRIPTION
**Pull request by tombaeyens**

### CODE REVIEW

File: soda/core/soda/execution/change_over_time_metric_check.py [hunk 1]
```diff
@@ -45,6 +45,17 @@ def evaluate(self, metrics: Dict[str, Metric], historic_values: Dict[str, object
             else:
                 self.check_value = float(metric_value) - float(historic_value)
 
+            historic_descriptor = self.historic_descriptors[KEY_HISTORIC_METRIC_AGGREGATE]
+            if isinstance(historic_descriptor, HistoricChangeOverTimeDescriptor):
+                if historic_descriptor.change_over_time_cfg.percent:
+                    if historic_value == 0:
+                        if self.check_value == 0:
```
Comment (vijaykiran) [inline soda/core/soda/execution/change_over_time_metric_check.py:52]: I am not sure what this if/else is doing .. why set it to 0 if it is 0?

**Review by vijaykiran — CHANGES_REQUESTED** [review]

File: soda/core/soda/execution/change_over_time_metric_check.py (hunk 1 above)
Comment (vijaykiran) [reply soda/core/soda/execution/change_over_time_metric_check.py:52]: @tombaeyens  can you check this ?

Comment (tombaeyens) [reply soda/core/soda/execution/change_over_time_metric_check.py:52]: It's if the metric value evolves from 0 to 0, the change should be 0 %.  If the metric value changes from 0 to any other number there is an infinite change %, which was decided to make it not evaluated.  (still think there should be a better solution for this.)

**Comment by tombaeyens** [comment]
This PR is related to docs PR https://github.com/sodadata/docs/pull/308

**Pull request merged**